docs/requirements.txt

```diff
@@ -0,0 +1,8 @@
+# Notebook dependencies
```

Comment:
I'd prefer to have no `requirements.txt` files whatsoever. We should be able to handle this with poetry.